CONTRIBUTING.md

```diff
@@ -2,29 +2,57 @@
 
-- Fork the repository on GitHub
+- Fork and clone the repository on GitHub.
+  -  This project uses [Go Modules](https://blog.golang.org/using-go-modules) and does not need to be cloned inside the `$GOPATH` provided that your installed version of GO is greater than or equal to 1.11.
+- Review the design documentation in the [doc/design](./doc/design) directory
+- Watch the content on the [Operator Framework Youtube Channel][of-youtube]
 - See the [developer guide](./DEVELOPMENT.md) for build instructions
+- Join the existing [Communication][communication] channels and introduce yourself to the other OLM Developers
 
 ## Reporting bugs and creating issues
 
-Reporting bugs is one of the best ways to contribute. However, a good bug report has some very specific qualities, so please read over our short document on [reporting bugs](./doc/dev/reporting_bugs.md) before submitting a bug report. This document might contain links to known issues, another good reason to take a look there before reporting a bug.
+If any part of the operator-lifecycle-manager project has bugs or documentation mistakes, please let us know by [opening an issue][operator-olm-issue]. We treat bugs very seriously and believe no issue is too small. Before creating a bug report, please check that an issue reporting the same problem does not already exist. If you aren't sure if the observed behavior is a bug, please reach out to the OLM team using the links provided under [communication][communication].
+
+To make the bug report accurate and easy to understand, please try to create bug reports that are:
+
+- Specific. Include as much detail as possible: which version, what environment, what configuration, etc.
+
+- Reproducible. Include the steps to reproduce the problem. We understand some issues might be hard to reproduce, please include the steps that might lead to the problem.
+
+- Isolated. Please try to isolate and reproduce the bug with minimum dependencies. It would significantly slow down the speed to fix a bug if too many dependencies are involved in a bug report. Debugging external systems that rely on operator-lifecycle-manager is out of scope, but we are happy to provide guidance in the right direction or help with using operator-lifecycle-manager itself.
+
+- Unique. Do not duplicate existing bug report.
+
+- Scoped. One bug per report. Do not follow up with another bug inside one report.
+
+It may be worthwhile to read [Elika Etemad’s article on filing good bug reports][filing-good-bugs] before creating a bug report.
+
+We might ask for further information to locate a bug. A duplicated bug report will be closed.
 
 ## Contribution flow
 
 This is a rough outline of what a contributor's workflow looks like:
 
 - Create a topic branch from where to base the contribution. This is usually master.
-- Make commits of logical units.
-- Make sure commit messages are in the proper format (see below).
+- Make commits of logical units using the `--signoff` flag.
```

Comment:
Would it make more sense to call out that we require commits to be signed off (enforced through a DCO check) vs. explicitly specifying you need to use the `--signoff` flag?